sdk/identity/identity-cache-persistence/package.json

```diff
@@ -60,7 +60,8 @@
   "sideEffects": false,
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure/identity": "^2.1.0",
+    "@azure/identity": "^3.1.3",
+    "@azure/identity-common": "1.0.0-beta.1",
```

Comment:
Is the plan that we'll ship a stable version of identity-common before we ship a new version of identity that depends on it?

Reply:
yes

Reply:
should we just set the common version to `1.0.0` now then? is there a reason to start with beta.1?

Reply:
We can start with 1.0.0. No reason to keep it beta